client/js/power-zone.js

```diff
@@ -63,7 +65,7 @@ function renderZoneCards(numZones) {
 
     // Recommended Power
     const recPowerFormGroup = document.createElement('div');
-    recPowerFormGroup.className = 'form-group col-md-4';
+    recPowerFormGroup.className = 'form-group col-lg-4';
```

Comment:
To be honest, I think this is harder to read on medium sized screens, not easier, as each row looks pretty much the same. If we did do this I think labeling the fields would be really good, but was there an issue with having them all on the same line?

Reply:
At some point it was hard to read the placeholders for these inputs. 
I will take a screenshot later.

Reply:
Note how you can't see Recommended Power? Another alternative is to have the placeholder say "Rec. Power"
<img width="1058" alt="Screen Shot 2019-08-23 at 8 27 48 pm" src="https://user-images.githubusercontent.com/18709969/63586369-8076de80-c5e4-11e9-9188-939c7f67658c.png">

Reply:
I see what you mean... When there's values filled in though you can't see any sort of label at all so it's even harder to read. I think using Rec. Power for now would be easier to read, but maybe eventually this could be inputed using a table or something.

Up to you then :)

Reply:
I will leave it as Recommended Power for now (Just in case people aren't sure what Rec. means). I do agree that this is probably not the best way of accomplishing this. Good suggestion for a table!